dependency/dependency_test.go

```diff
@@ -149,6 +149,16 @@ func (v *vaultServer) CreateSecret(path string, data map[string]interface{},
 	return err
 }
 
+// deleteSecret lets us delete keys as needed for tests
+func (v *vaultServer) deleteSecret(path string) error {
+	path = v.secretsPath + "/" + path
```

Comment:
maybe better not to modify the input? maybe instantiate a new variable? or else maybe all on one line like:
```
/* blah */.Delete(v.secretsPath + "/" + path)
```

Reply:
No variable at all is better IMO.  Thanks!